DigitalOffice/Services/CheckRightsService/CheckRightsService.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="5.0.0-preview.6.20312.4" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="5.0.0-preview.6.20312.4" />
```

Comment:
Is it a good practise to use preview versions of the packages?

Reply:
I think not. 5.0 update will be released only in September and we need to work on the stable version.
Also, how is this even possible? NuGet prevents this kind of versioning and allows to install only stable versions.
![image](https://user-images.githubusercontent.com/61093487/87470543-58e29780-c636-11ea-958e-bfb526f40c88.png)

Reply:
> I think not. 5.0 update will be released only in September and we need to work on the stable version.
> 
> Also, how is this even possible? NuGet prevents this kind of versioning and allows to install only stable versions.
> 
> ![image](https://user-images.githubusercontent.com/61093487/87470543-58e29780-c636-11ea-958e-bfb526f40c88.png)
> 
> 

Yikes. No problem changing it, but usually previews which are already accessible r ok, at least haven't heard of possible "disasters" caused by using them:) However, might be mistaken:)